packages/admin/admin/src/tabs/RouterTabs.tsx

```diff
@@ -88,20 +130,20 @@ function RouterTabsComponent({
     let foundFirstMatch = false;
 
     return (
-        <div className={classes.root}>
+        <Root {...slotProps?.root}>
```

Comment:
The `sx` and `className` props should be passed into the `Root`, you can probably do it with `...restProps`.